### DISCUSSION TITLE
Not seeing my other personal account

### DISCUSSION THREAD

**Discussion by aleksandar-b** [vercel/community#4257] [Account Issue] [answered]
### Question

I don't see anymore my other personal account. I had several projects and domains on it so this is a problem for me.

**Comment by amyegan** [comment, marked as answer]
Please follow the instructions shared in https://github.com/vercel/community/discussions/4263#discussioncomment-7123763 🙏

**Reply by aleksandar-b** [reply]
I did that several times..

**Reply by amyegan** [reply]
If you let me know the case number, I can check on it for you 🙇
